KLSadd_insertion/src/updateChapters.py

```diff
@@ -187,162 +468,197 @@ def referencePlacer(chap, references, p, kls, refII, refIII, specref):
+                    chap_nums.append(9)
+                    name = word[word.find("{") + 1: word.find("}")]
+                    math_people.append(name + "#")
+                    specref9.append(index-1)
+                if "14." in word:
+                    chap_nums.append(14)
+                    name = word[word.find("{") + 1: word.find("}")]
+                    math_people.append(name + "#")
+                    if len(specref14) == 0:
+                        specref9.append(index - 1)
+                    specref14.append(index - 1)
+                indexes.append(index-1)
+            if "paragraph{" in word and index > 313:
+                klsaddparas.append(index-1)
+            if "subsub" in word and index > 313:
+                klsaddparas.append(index - 1)
+            if "\subsection*{" in word and index > 313:
+                klsaddparas.append(index - 1)
+        klsaddparas.append(2246)
+        print(indexes)
+        # now indexes holds all of the places there is a section
+        # using these indexes, get all of the words in between and add that to the paras[]
+        for i in range(len(indexes)-1):
+            box = ''.join(addendum[indexes[i]: indexes[i+1]-1])
+            paras.append(box)
+            print(box)
+        box2 = ''.join(addendum[indexes[35]: 2245])
+        paras.append(box2)
+
+
+        # paras now holds the paragraphs that need to go into the chapter files, but they need to go in the appropriate
+        # section(like Wilson, Racah, Hahn, etc.) so we use the mathPeople variable
+        # we can use the section names to place the relevant paragraphs in the right place
+
+        # as of 2/8/16 the paragraphs will go before the References paragraph of the relevant section
+        # parse both files 9 and 14 as strings
+
+        # chapter 9
+        with open(DATA_DIR + "chap09.tex", "r") as ch9:
+            entire9 = ch9.readlines()  # reads in as a list of strings
+
+        # chapter 14
+        with open(DATA_DIR + "chap14.tex", "r") as ch14:
+            entire14 = ch14.readlines()
+
+        # call the findReferences method to find the index of the References paragraph in the two file strings
+        # two variables for the references lists one for chapter 9 one for chapter 14
+        chapticker = 0
+        references9 = find_references(entire9, chapticker)
+        chapticker += 1
+        references14 = find_references(entire14, chapticker)
+        ref14_3.insert(88, 1217)
+        ref14_3.insert(244, 3053)
+        ref14_3.insert(198, 2554)
+
+        # call the fixChapter method to get a list with the addendum paragraphs added in
+        chapticker2 = 0
+        str9 = ''.join(fix_chapter(entire9, references9, paras, addendum, kls_list_all, chapticker2))
+
+        chapticker2 += 1
+        str14 = ''.join(fix_chapter(entire14, references14, paras, addendum, kls_list_all, chapticker2))
+
+    # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
I think there is python style to put lines in comments that is different than this